### ISSUE TITLE
Complete or remove TODOs in PCB build instructions

### ISSUE THREAD

**Issue by Achllle** [nasa-jpl/open-source-rover#444]
There are 13 TODOs in the [PCB build instructions](https://github.com/nasa-jpl/open-source-rover/tree/master/electrical/pcb#33-solder-resistor-r1-and-power-diode-d1) currently. They should either be completed or moved to an issue.

**Comment by knickels** [comment]
I looked at this today, and see 3 TODOs.  I submitted a PR for one of them #551 #552 

I am confused by the TODO in Step 2:
>  TODO: add versions for v2.0.3
I don't see a version number in KiCAD.  Is this just to pull the currrent schematics up in KiCAD and generate a new PDF?

**Comment by Achllle** [comment]
I believe this refers to the pictures of the control boards since those are likely from an earlier version, v2.0.1. If you or anyone else has pictures, it could avoid confusion for those not aware of previous versions.

**Comment by knickels** [comment]
No, I think it's talking about the PDF of the schematic.

**Comment by Achllle** [comment]
You're right. Indeed that documentation is simply out of date. Updating those would be the approach, but I would also be ok with just removing those files since installing KiCAD is not that hard and free.

**Comment by knickels** [comment]
cool, done.
